src/Uno.UI/UI/Xaml/IFrameworkElementImplementation.macOS.tt

```diff
@@ -576,38 +576,49 @@ namespace <#= mixin.NamespaceName #>
 
 		#region IsEnabled Dependency Property
 
+		// public bool IsEnabled
+		// {
+		// 	get { return (bool)this.GetValue(IsEnabledProperty); }
+		// 	set { this.SetValue(IsEnabledProperty, value); }
+		// }
+
```

Comment:
Commented (generated) code

Reply:
Well ... because when we add some commented code, why should we limit us to only one place ?